src/bindings/python/tests/test_graph/test_ops_scatter_nd_update.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+
+from openvino.runtime import PartialShape, Type
```

Comment:
```suggestion
from openvino import PartialShape, Type
```

Reply:
sometime  ago we introduced new imports, so it'd be better to use new namespace :)